dkist/dataset/tiled_dataset.py

```diff
@@ -174,12 +188,20 @@ def plot(self, slice_index: int, share_zscale=False, **kwargs):
         if share_zscale:
             for ax in fig.get_axes():
                 ax.get_images()[0].set_clim(vmin, vmax)
-        timestamp = self[0, 0].axis_world_coords("time")[-1].iso[slice_index]
-        fig.suptitle(f"{self.inventory['instrumentName']} Dataset ({self.inventory['datasetId']}) at time {timestamp} (slice={slice_index})", y=0.95)
+        title = f"{self.inventory['instrumentName']} Dataset ({self.inventory['datasetId']}) at "
+        for i, (coord, val) in enumerate(tiles[0].global_coords.items()):
+            if coord == "time":
+                val = val.iso
+            title += f"{coord} {val}" + (", " if i != len(slice_index)-1 else " ")
+        title += f"(slice={(slice_index if len(slice_index) > 1 else slice_index[0])})".replace("slice(None, None, None)", ":")
+        fig.suptitle(title, y=0.95)
         return fig
 
     @property
     def slice_tiles(self):
+        """
+        Returns a new TiledDataset with the given slice applied to each of the tiles.
```

Comment:
I think this needs an example and some more info. It's not obvious from this what the syntax is.